natspec-smells.config.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * List of supported options: https://github.com/defi-wonderland/natspec-smells?tab=readme-ov-file#options
```

Comment:
Observation: `constructorNatspec` option is by default set to false. 🤔

Reply:
@dristpunk let's see if most projects you have been testing the tool with have constructor natspec. I argue it's not that popular and hence should be `false` even in our own projects.

Reply:
we decided to leave it as it is for now